scripts/npm/publish.sh

```diff
@@ -4,4 +4,4 @@ echo "registry=https://registry.npmjs.org/
 //registry.npmjs.org/:_authToken=$NPM_AUTH_TOKEN
 access=public" > ~/.npmrc
 
-yarn oao all -i sandbox "yarn publish --tag $NPM_TAG --access public"
+yarn workspaces foreach exec "yarn npm publish --tag $NPM_TAG --access public"
```

Comment:
missing ignored sandbox (`-i sandbox` in previous), it will probably fail because sandbox is private

Reply:
or we can probably use `--no-private` like in interface
https://github.com/contember/interface/blob/20d6739498f94391c8e68578e0ba6a5036a3ee52/.github/workflows/ci.yaml#L213

Reply:
Yep `-pt --no-private` added to make sure we are trying to publish only no-private packages